pytest_jira.py

```diff
@@ -113,8 +129,8 @@ def is_affected(self, issue_id):
         else return False
         """
         return (
-            self._affected_version(issue_id) and
-            self._affected_components(issue_id)
+                self._affected_version(issue_id) and
```

Comment:
I believe this is an unnecessary indent.

Reply:
Yeah, pycharm likes it that way for some reason.

Reply:
It is back :-D